iOS/Flows/Conversation/ConversationViewController.swift

```diff
@@ -38,7 +38,11 @@ class ConversationViewController: FullScreenViewController,
     var didTapConversationTitle: CompletionOptional = nil
 
     // Custom Input Accessory View
-    lazy var messageInputAccessoryView = ConversationInputAccessoryView(with: self)
+    lazy var messageInputAccessoryView: ConversationInputAccessoryView = {
+        let view: ConversationInputAccessoryView = ConversationInputAccessoryView.fromNib()
+        view.delegate = self
+        return view
+    }()
```

Comment:
The input view is now loaded from a xib.

Reply:
nice